app/components/ui/LabelTag.tsx

```diff
@@ -2,18 +2,26 @@ import React from "react";
 import classNames from "classnames";
 import styles from "./LabelTag.module.scss";
 
-interface LabelTagProps {
-  label: string;
-  withTooltip?: boolean;
-}
+// This component relies on the shape and naming conventions of data Shelter
+// Tech syncs to Algolia which distinguishes service and organization results
+// with a "type" field.
+//
+// CAVEAT: It's unlikely this will change without our knowledge but developers
+// should be aware of the possibility.
+const SHELTER_TECH_TYPE_FOR_ORGANIZATION = "resource";
```

Comment:
**nitpick (non-blocking):** This was reusable before and now it is specific to the resource card so I'd suggest renaming the component accordingly